tests/resources/Util.robot

```diff
@@ -423,6 +433,7 @@ Run Regression Tests
     ${rc}  ${output}=  Run And Return Rc and Output  curl -sk ${vic-admin}/container-logs.tar.gz | tar tvzf -
     Should Be Equal As Integers  ${rc}  0
     Log  ${output}
+    Should Contain  ${output}  ${container}/output.log
```

Comment:
output.log isn't collected currently - it's application level output and really shouldn't appear in a support bundle normally.
I did put a comment for @matthewavery into #2669 about gathering it when `debug > 2` so that we have a mechanism during tests, but that comes with a behavioural change in the code. Which we should be testing, but it's not the default deployment configuration.